src/index.ts

```diff
@@ -30,12 +32,20 @@ export function load(app: Application) {
         type: ParameterType.Boolean,
     });
 
+    app.options.addDeclaration({
+        name: "additionalModuleSources",
+        help: "[typedoc-plugin-mdn-links]: Additional module sources to resolve.",
+        type: ParameterType.Array,
+        defaultValue: [],
+    });
+
     const resolvers = [
         resolveGlobalName,
         resolveDomName,
         resolveCssName,
         resolveCanvasName,
         resolveWebAudioName,
+        resolveWebApiPath,
```

Comment:
Do we need any of the other resolvers anymore? I kind of expected the web API resolver to work for all of these links, so they wouldn't need to be hardcoded anymore.

Reply:
Yep, I would like to check through and make sure that there are no regressions before cutting out the other resolvers though and figured it would be better to do that after an initial review.